repos/arcade.proj

```diff
@@ -7,6 +7,7 @@
     <BuildCommandArgs />
     <BuildCommandArgs>$(BuildCommandArgs) $(FlagParameterPrefix)pack</BuildCommandArgs>
     <BuildCommandArgs>$(BuildCommandArgs) $(FlagParameterPrefix)configuration $(Configuration)</BuildCommandArgs>
+    <BuildCommandArgs>$(BuildCommandArgs) /p:IncludeTestUtils=true</BuildCommandArgs>
```

Comment:
What is this flag? Should it be conditional on building tests?  Does it bring in prebuilts?

Reply:
That's true. And it doesn't bring in any additional prebuilts.

Reply:
I'm not sure Arcade sure care about this at all actually.  Why is this needed?